spec/dmv_spec.rb

```diff
@@ -36,6 +36,7 @@
       @dmv.add_facility(@facility_1)
       @dmv.add_facility(@facility_2)
       @dmv.add_facility(@facility_3)
+      # require 'pry'; binding.pry
```

Comment:
I would make sure to remove all of these before you push and merge to main. You don't want to do that when you get on the job.